docs/advanced_tutorial/customize_homepage.md

```diff
@@ -71,7 +71,7 @@ class HomePage(Page):
 
 You might already be familiar with the different parts of your `HomePage` model. The `image` field is a `ForeignKey` referencing Wagtail's built-in Image model for storing images. Similarly, `hero_cta_link` is a `ForeignKey` to `wagtailcore.Page`. The `wagtailcore.Page` is the base class for all other page types in Wagtail. This means all Wagtail pages inherit from `wagtailcore.Page`. For instance, your `class HomePage(page)` inherits from `wagtailcore.Page`.
 
-Using `on_delete=models.SET_NULL` ensures that if you remove an image or hero link from your admin interface, the `image` and `hero_cta_link` fields on your Homepage will be set to null, preserving their data entries. Read the [Django documentation](https://docs.djangoproject.com/en/4.2/ref/models/fields/#django.db.models.ForeignKey.on_delete) for more values for the `on-delete` attribute.
+Using `on_delete=models.SET_NULL` ensures that if you remove an image or hero link from your admin interface, the `image` and `hero_cta_link` fields on your Homepage will be set to null, preserving their data entries. Read the [Django documentation](https://docs.djangoproject.com/en/4.2/ref/models/fields/#django.db.models.ForeignKey.on_delete) for more values for the `on_delete` attribute.
```

Comment:
Could we use an intersphinx link instead of a hard-coded one?

```suggestion
Using `on_delete=models.SET_NULL` ensures that if you remove an image or hero link from your admin interface, the `image` and `hero_cta_link` fields on your Homepage will be set to null, preserving their data entries. Read the [Django documentation](django.db.models.ForeignKey.on_delete) for more details on the `on_delete` attribute.
```

Also, I'm slightly confused by the "will be set to null, preserving their data entries" part. My immediate thought was, "if they will be set to null, why (or what) is that 'preserving'?"

I think we could make this clearer by saying something like e.g. "if the specified image or hero link is deleted via the admin interface, the corresponding fields in the `HomePage` instance will be set to null, but the `HomePage` instance (and data in other fields) will be preserved."

Reply:
Also, "Django documentation" isn't great as link text - ideally link text should work as a description of the link when read in isolation (for example, screen readers will do this when tabbing over links). I'd suggest:

```Read the [Django documentation on the `on_delete` attribute](django.db.models.ForeignKey.on_delete) for more details.```

Reply:
Thanks! Will update.